modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -204,12 +204,15 @@ struct GAPI_EXPORTS GMatDesc
 static inline GMatDesc empty_gmat_desc() { return GMatDesc{-1,-1,{-1,-1}}; }
 
 namespace gapi { namespace detail {
-// Checks GMatDesc fields if the passed matrix is a set of n-dimentional points of the given depth.
+// Checks GMatDesc fields if the passed matrix is a set of n-dimentional points of the given depth
+// and returns ture or false.
 // If n <= 0 given, the function counts the dimentionality by itself.
-// If ddepth < 0 given, there is no assert on the depth of the passed matrix
-// Returns two ints: quantity of the elements and their dimentionality.
-// In case input matrix can't be described as vector of points, returns (-1,-1).
-std::tuple<int,int> checkVector(const GMatDesc& in, const int n = -1, const int expectedDepth = -1);
+// If ddepth < 0 given, there is no assert on the depth of the passed matrix.
+// Outputs are quantity of the elements and their dimentionality.
```

Comment:
return value is not described

Reply:
There was a description, but really short
Description expanded